ai-service/tests/conftest.py

```
@@ -1,48 +1,50 @@
-        db_test_collection.delete(ids=ids)
+    # Clean up after each test - use the real get_collection() function
+    from ai_service.db_setup import get_collection
+
+    try:
+        collection = get_collection()
+        ids = collection.peek()["ids"]
+        if ids:
+            collection.delete(ids=ids)
+    except Exception:
+        # If collection doesn't exist or other error, just continue
+        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)